### ISSUE TITLE
Suggestion: Highlight End Portal Zealots in the End.

### ISSUE THREAD

**Issue by SirNapkin1334** [BiscuitDevelopment/SkyblockAddons#167]
As you might know, in Patch 0.7.3, End Portal Frame Zealots were re-introduced, but with different mechanics. Now, whenever killing a Zealot, there is a small chance of spawning an End Portal Frame Zealot, which is accompanied by a sound and a chat message. Perhaps you could add something similar to Ender Chest Highlighting, but with Portal frames? Perhaps you could use the color #ff00ff, which is also neon-bright, but easily distinguished from neon green. Additionally, since there is a chat message, perhaps you could add a warning when you spawn one, that appears on screen like the Summoning Eye alert.

**Comment by Keagel** [comment]
Can you supply the text message that's sent when one spawns?

**Comment by SirNapkin1334** [comment]
Yes, indeed I can. The message is known to contain the following: "A special zealot has spawned" it is not known if there is any punctuation, so it'd be best to use regex. I will later today attempt to test it to see the message for myself. Additionally, the message is bolded and dark purple (the color that Epic armor has).

**Comment by superhize** [comment]
This is the message
![javaw_2019-10-28_01-04-35](https://user-images.githubusercontent.com/17607478/67645271-f3c00800-f927-11e9-9eb1-76157f128c03.png)

**Comment by ichbineinNerd** [comment]
I'll get on this, seems like a good first issue
